crypto_kem/ntrulpr1013/avx2/api.h

```diff
@@ -0,0 +1,16 @@
+#ifndef PQCLEAN_NTRULPR1013_AVX2_API_H
+#define PQCLEAN_NTRULPR1013_AVX2_API_H
+
+
+
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_ALGNAME "ntrulpr1013"
+
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_SECRETKEYBYTES 1773
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_PUBLICKEYBYTES 1455
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_CIPHERTEXTBYTES 1583
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_BYTES 32
+
+int PQCLEAN_NTRULPR1013_AVX2_crypto_kem_keypair(unsigned char *pk, unsigned char *sk);
+int PQCLEAN_NTRULPR1013_AVX2_crypto_kem_enc(unsigned char *c, unsigned char *k, const unsigned char *pk);
+int PQCLEAN_NTRULPR1013_AVX2_crypto_kem_dec(unsigned char *k, const unsigned char *c, const unsigned char *sk);
```

Comment:
Did we not specify these should be `uint8_t`?

Reply:
Indeed. I changed it. Thanks